cmd/on.go

```diff
@@ -68,6 +70,8 @@ func newOnCmd(h *pkg.Hctl, out io.Writer) *cobra.Command {
 	}
 
 	cmd.PersistentFlags().StringVarP(&brightness, "brightness", "b", "", "Set brightness")
+	cmd.PersistentFlags().StringVarP(&color, "color", "c", "", "Set RGB color in format R,G,B")
+	cmd.PersistentFlags().IntVarP(&colorTemp, "color_temp", "t", 0, "Set color temperature in mireds (153-500)")
```

Comment:
```suggestion
	cmd.PersistentFlags().IntVarP(&colorTemp, "color-temp", "t", 0, "Set color temperature in mireds (153-500)")
```
let's stick to the more commonly used convention of using hyphens for word separation in long options